src/include/daos/object.h

```diff
@@ -27,6 +27,13 @@
 #include <daos/tse.h>
 #include <daos_obj.h>
 
+/* EC parity is stored in a private address range that is selected by setting
+ * the most-significant bit of the offset (an unsigned long). This effectively
+ * limits the addressing of user extents to the lower 63 bits of the offset
+ * range.
+ */
+#define DAOS_EC_PARITY_BIT	(1ULL << 63)
```

Comment:
why define a new variable, instead of using PARITY_INDICATOR?

Reply:
the naming of "PARITY_INDICATOR" without good prefix seems not good to put in  daos/object.h, actually i think can replay previous usage of PARITY_INDICATOR to DAOS_EC_PARITY_BIT.